dali/kernels/imgproc/convolution/convolution_gpu_test.cu

```diff
@@ -144,19 +146,19 @@ struct ConvolutionGpuKernelTest : public ::testing::Test {
 
 TYPED_TEST_SUITE_P(ConvolutionGpuKernelTest);
 
-  // ndim, has_channels, convolution axis, window size, input type
+  // ndim, has_channels, convolution axis, window size, input type, [output type = float]
 using ConvolutionTestValues = ::testing::Types<
     // 1D
-    convolution_params<1, false, 0, 3, float>,
-    convolution_params<1, false, 0, 15, float>,
-    convolution_params<1, false, 0, 51, float>,
-     convolution_params<1, false, 0, 101, float>,
+    convolution_params<1, false, 0, 3, int16_t, int16_t>,
```

Comment:
Why replacing the old test instead of extending them?

Reply:
I'm hitting the limit of how many I can specify :P

Reply:
I made it a bit shorter by testing those windows in one go.